src/sentry/web/frontend/setup_wizard.py

```diff
@@ -64,16 +64,19 @@ def get(self, request: HttpRequest, wizard_hash) -> HttpResponseBase:
         Redirects to organization whenever cache has been deleted
         """
         context = {"hash": wizard_hash, "enableProjectSelection": False}
-        key = f"{SETUP_WIZARD_CACHE_KEY}{wizard_hash}"
+        cache_key = f"{SETUP_WIZARD_CACHE_KEY}{wizard_hash}"
+
+        org_slug = request.GET.get("org_slug", None)
+        project_slug = request.GET.get("project_slug", None)
```

Comment:
Just for my understanding: `dict.get` second argument is `None` by default, is it intentional to make it explicit?

Reply:
just force of habit, will remove it 👍